packages/atlas/src/joystream-lib/extrinsics.ts

```diff
@@ -71,7 +71,7 @@ type PublicExtrinsic<TxFunction, ReturnValue> = TxFunction extends (...a: infer
   ? (...a: [...U, ExtrinsicStatusCallbackFn | undefined]) => Promise<ReturnValue>
   : never
 
-const PERMILLS_PER_PERCENTAGE = 10
+const PERMILLS_PER_PERCENTAGE = 10_000
```

Comment:
Wow so permill does mean "per million" and not the [`‰` permil](https://en.wikipedia.org/wiki/Per_mille). That's really a bad abbreviation choice on substrate!